server/src/code-runner/docker-runner.ts

```diff
@@ -48,14 +73,23 @@ export class DockerRunner implements CodeRunner {
 
     const mounts = flatMap(configuration.mountPoints, mp => ['-v', `${mp.source}:${mp.destination}:ro`]);
 
+    let mode = ['--read-only'];
+
+    if (environment.writeable) {
```

Comment:
How do you feel about making this a ternary?

```
let mode = environment.writeable ? [...] : [...] 
```

My reasoning is that a statement based solution makes it easy to overlook that this `mode` is really one thing or the other whereas the ternary based (aka expression based) solution makes that impossible. It's a bit of a nitpick. Feel free to ignore if you disagree.

Reply:
Done :)!